0-bootstrap/main.tf

```diff
@@ -54,16 +54,29 @@ module "seed_bootstrap" {
   sa_enable_impersonation = true
 }
 
-module "cloudbuild_bootstrap" {
-  source                  = "terraform-google-modules/bootstrap/google//modules/cloudbuild"
-  version                 = "~> 1.0"
-  org_id                  = var.org_id
-  folder_id               = google_folder.seed.id
-  billing_account         = var.billing_account
-  group_org_admins        = var.group_org_admins
-  default_region          = var.default_region
-  terraform_sa_email      = module.seed_bootstrap.terraform_sa_email
-  terraform_sa_name       = module.seed_bootstrap.terraform_sa_name
-  terraform_state_bucket  = module.seed_bootstrap.gcs_bucket_tfstate
-  sa_enable_impersonation = true
+// TODO(caleonardo / bharathkkb): Choose between cloudbuild_bootstrap and jenkins_bootstrap with a variable
```

Comment:
I don't think this should be a variable, it should just be a separate sub-module as it is currently.

Reply:
I agree, plsu with TF 0.12 we can't technically do conditional module instantiation unless we do something like [this](https://github.com/terraform-google-modules/terraform-google-gcloud/blob/a3aad69cca8e0f25ad926dd63008e884f3007aff/variables.tf#L17).

Reply:
Thank you - I've added an "enabled" variable for the Jenkins sub-module. In a future PR, we can add the same to the cloudbuild module if that is what we agreed on.

Reply:
After talking with @rjerrems, decided to remove the "enabled" variable, we will keep it simple for now. if people do not want to use the jenkins_bootstrap, they should just delete / comment the module and use cloudbuild_bootstrap instead.